Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java

```diff
@@ -251,13 +248,13 @@ public void handleActivityResult(@ActivityRequestCode int requestCode, int resul
 				Uri selectedGalleryImageUri = data.getData();
 				setTool(ToolType.IMPORTPNG);
 				toolController.switchTool(ToolType.IMPORTPNG);
-				interactor.loadFile(this, LOAD_IMAGE_IMPORTPNG, maxWidth, maxHeight, selectedGalleryImageUri);
+				interactor.loadFile(this, LOAD_IMAGE_IMPORTPNG, selectedGalleryImageUri);
```

Comment:
i think the whole ticket can be reduced to an `if(openedFromCatroid)...else` here, because if you remove `loadFile(...)` with `maxWidth` and `maxHeight` completely importing pictures will also never get scaled when opened from catroid.

Reply:
isnt scaling when importing from catroid already deactivated?

Reply:
does not look like that to me.